nutype_macros/src/common/models.rs

```diff
@@ -200,7 +200,6 @@ pub struct RawGuard<Sanitizer, Validator> {
 
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub enum DeriveTrait {
-    Asterisk,
     Normal(NormalDeriveTrait),
 }
```

Comment:
Since we don't have `DeriveTrait::Asterisk` anymore, there is not point to have `DeriveTrait` with only one variant `Normal`.

Could you please:
* Remove `DeriveTrait`
* Rename `NormalDeriveTrait` -> `DeriveTrait`
* Address also the compile errors

Reply:
All of the above addressed with 77a99b3.